docs/solutions/microsoft-365-scenarios.md

```diff
@@ -32,6 +32,7 @@ Microsoft 365 is a suite of apps that help you stay connected and get things don
 
 - [Word](https://www.microsoft.com/microsoft-365/word). Create documents and improve your writing with built-in intelligent features.
 - [Excel](https://www.microsoft.com/microsoft-365/excel). Simplify complex data and create easy-to-read spreadsheets.
+- [Microsoft Graph] (https://learn.microsoft.com/en-us/graph/integration-patterns-overview). Learn about integration patterns that you can apply to solutions that you can build using Microsoft Graph. 
```

Comment:
<https://www.microsoft.com/en-us/microsoft-365/products-apps-services> doesn't list this as an app or service of M365.  The list in this doc is preceded with: "Microsoft 365 is a suite of apps that help you stay connected and get things done. It includes these tools:"

Instead of adding it here, can instead you add it right below this list as a standalone sentence saying something like "API access to Microsoft 365 apps and services is facilitated through [Microsoft Graph](/graph/integration-patterns-overview).